server/sonar-server/src/main/java/org/sonar/server/computation/measure/Measure.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.computation.measure;
+
+import com.google.common.base.Optional;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+
+public interface Measure {
+
+  enum ValueType {
+    NO_VALUE, BOOLEAN, INT, LONG, DOUBLE, STRING
+  }
+
+  enum AlertStatus {
+    OK("Green"),
+    WARN("Orange"),
+    ERROR("Red");
+
+    private final String colorName;
+
+    AlertStatus(String colorName) {
```

Comment:
The term "alert" should be removed. I suggest "QualityGateStatus" or "GateStatus"